src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithoutStrongNames.txt

```diff
@@ -19,15 +19,15 @@
 #
 ################################
 
-# Base directory: C:\Code\sonarlint-visualstudio\src\Integration.Vsix\bin\Debug\VS2022\net472
+# Base directory: D:\repos\sq\slvs\src\Integration.Vsix\bin\Debug\VS2022\net472
```

Comment:
Same here

Reply:
Good spot. It doesn't matter, however - the AsmRef tool doesn't check comments.